lib/errors.js

```diff
@@ -109,7 +108,8 @@ exports.Err = class {
 
         return format.replace(/\{\{(\!?)([^}]+)\}\}/g, ($0, isSecure, name) => {
 
-            const value = Hoek.reach(this.context, name);
+            const context = Object.assign({}, this.context, { key: this.context.label });
```

Comment:
@Marsup I'm not breaking it, I'm making it work the way it was working.
This PR fixes the issue with `err.context.key` being the label and adds `err.context.label`.
I can see now that the template language also has the bug of the key being the label, but it had that bug before I contributed my changes.
I think that the bug with the template language is a different issue and it could be fixed separately from this one.